README.md

```diff
@@ -1,2 +1,112 @@
-# pupmod-simp-simp_nfs
-NFS Profile Module for SIMP
+[![License](http://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html) [![Build Status](https://travis-ci.org/simp/pupmod-simp-simp_nfs.svg)](https://travis-ci.org/simp/pupmod-simp-simp_nfs) [![SIMP compatibility](https://img.shields.io/badge/SIMP%20compatibility-6.*-orange.svg)](https://img.shields.io/badge/SIMP%20compatibility-6.*-orange.svg)
+
+#### Table of Contents
+
+1. [Description](#description)
+2. [Setup - The basics of getting started with simp_nfs](#setup)
+    * [What simp_nfs affects](#what-simp_nfs-affects)
+3. [Usage - Configuration options and additional functionality](#usage)
+4. [Reference - An under-the-hood peek at what the module is doing and how](#reference)
+5. [Limitations - OS compatibility, etc.](#limitations)
+6. [Development - Guide for contributing to the module](#development)
+    * [Acceptance Tests - Beaker env variables](#acceptance-tests)
+
+## Description
+
+This module is a [SIMP](https://simp-project.com) Puppet profile for setting up
+common NFS configurations as supported by the SIMP ecosystem
+
+### This is a SIMP module
+
+This module is a component of the [System Integrity Management Platform](https://github.com/NationalSecurityAgency/SIMP), a
+compliance-management framework built on Puppet.
+
+If you find any issues, they may be submitted to our [bug tracker](https://simp-project.atlassian.net/).
+
+This module is optimally designed for use within a larger SIMP ecosystem, but
+it can be used independently:
+
+ * When included within the SIMP ecosystem, security compliance settings will
+   be managed from the Puppet server.
+ * If used independently, all SIMP-managed security subsystems are disabled by
+   default and must be explicitly opted into by administrators.  Please review
+   the parameters in
+   [`simp/simp_options`](https://github.com/simp/pupmod-simp-simp_options) for
+   details.
+
+## Setup
+
+### What simp_nfs affects
+
+This module provides commonly used configurations for NFS server and client
+systems.
+
+## Usage
+
+### Serve NFS Home Directories over Stunnel
```

Comment:
@jeefberkey Because that's the one that works with local logins. KRB5 isn't done yet.